packages/mendel-pipeline/src/client/generators.js

```diff
@@ -43,6 +49,8 @@ class MendelGenerators {
                 `${resultBundle.entries.size} entries for`,
                 `bundle, "${bundle.options.id}"`,
             ].join(' '));
+
+            this.table.push([bundle.options.id, bundle.options.generator, resultBundle.entries.size]);
```

Comment:
i kinda wish we didn't push if the option is off (like can we do this later in L73?)